src/redis-cli.c

```diff
@@ -2607,6 +2609,17 @@ static int isSensitiveCommand(int argc, char **argv) {
                 return 0;
             }
         }
+    } else if (argc > 4 && !strcasecmp(argv[0], "sentinel")) {
+        /* SENTINEL CONFIG SET sentinel-pass */
+        if (!strcasecmp(argv[1], "config") && !strcasecmp(argv[2], "set") && !strcasecmp(argv[3], "sentinel-pass")) {
```

Comment:
Is sentinel-user need hiddened?

Reply:
Could you please give me more so I can understand why you think it is better to hide?

Reply:
IMHO, it is useful for debugging to have (only) username being recorded. 
Maybe we should consider avoid file `rediscli_history` by default as it spam for the common case and might be even bigger security vulnerability than only whether or not having the user since it is data-at-rest on client side entities which usually less secure (@oranagra).

Reply:
i think that if this feature will be disabled by default, nearly no one will enable it.
i'm not very familiar with sentinel, so maybe i don't understand sentinel-user, but i do see that in HELLO we redact both the username and password, so i think it's probably right to hide this config.
i suppose users won't usually be missing / needing that line.

Reply:
@oranagra , the file `.rediscli_history` is a generated file of redis-cli. Irrelevant to sentinel. Just because most of the clients aren't aware of it, no precaution is being made to protect this leftover file. (Many times the commands and their arguments tell much more interesting details, behind who is the user).

Reply:
In fact, I do not think sentinel-user is related to the safety. Password is more sensitive, @zeekling could you provide us more detail why you think sentinel-user should be hidden?

Reply:
re-reading this after long time (sorry).
@moticless i know this file is not specific to sentinel. my comment about it being bad idea to disable it by default meant both both sentinel and redis. (unless the suggestion was to somehow disable it by default only when connecting to sentinel? i don't think redis-cli knows that).

regarding the original discussion, again, if we choose to hide passwords, we should also hide user names (again unless i'm missing some sentinel specific detail about what's `sentinel-user`)

Reply:
I agree with disable it by default like what you said, but i need take time to think why username need to be hidded as well